index.html

```diff
@@ -8,49 +8,64 @@
         <link rel="stylesheet" href="css/index.css">
     </head>
 <body>
+<nav>
 
 <h1>Hi! My name is Morgan</h1>
-
 <section id="About">
     <h2>About</h2>
-        <li>Hi! Thank you for taking the time to want to learn more about me.</li>
-        <li>I'm currently studying for my MBA in Data Analytics at Franklin University in Columbus, Ohio, and I am also a Columbus native.</li>
-        <li>I enjoy reading fantasy, romance and non-fiction autobiographies, the arts, such as visiting museums
-    and participating in ceramics, oil painting and both pencil and charcoal drawing.</li>
-        <li>I'm 25, a Libra, my favorite colors are Black, Lavender and Red, and my favorite animals are Killer Whales,
-    Koalas, Cats and Sea Otters. OH! And I love Hello Kitty and Princess Diana.</li>
+        <li>Hi! Thank you for taking the time to want to learn more about me. I'm currently studying for my MBA in Data Analytics at Franklin</li>
+        <li>University in Columbus, Ohio, and I am also a Columbus native. I enjoy reading fantasy, romance and non-fiction autobiographies,</li>
+        <li>the arts, such as visiting museums and participating in ceramics, oil painting and both pencil and charcoal drawing. I'm 25, a Libra,</li>
+        <li> my favorite colors are Black, Lavender and Red, and my favorite animals are Killer Whales, Koalas, Cats and Sea Otters. OH! And</li>
```

Comment:
You don't need to use ` <li>` for About section, simply use `<p>`  to seprate the parts